test/e2e/utils/helpers.go

```diff
@@ -360,77 +205,54 @@ func WaitForNodeConfigState(ctx context.Context, ncClient scyllav1alpha1client.N
 		return allDone, nil
 	}
 
-	event, err := watchtools.UntilWithSync(ctx, lw, &scyllav1alpha1.NodeConfig{}, nil, func(event watch.Event) (bool, error) {
+	event, err := watchtools.UntilWithSync(ctx, lw, reflect.New(reflect.TypeOf(*new(ObjectPtrType)).Elem()).Interface().(runtime.Object), nil, func(event watch.Event) (bool, error) {
```

Comment:
```suggestion
	event, err := watchtools.UntilWithSync(ctx, lw, *new(ObjectPtrType), nil, func(event watch.Event) (bool, error) {
```

Reply:
to many iterations, you are right that the simple one works ;)

Reply:
fixed